webapp-client/src/stylesheets/Item.css

```diff
@@ -0,0 +1,22 @@
+.item {
+  margin: 15px;
+  border-radius: 3px;
+  background-color: blanchedalmond;
+}
+
+.highlight {
+  font-size: 25px;
+}
+
+a {
+  font-size: 18px;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.notes {
+  font-size: 20px;
+}
```

Comment:
Add newline

Reply:
I changed it since it's a small edit, but I forgot to mention (now added to the description of this PR) to ignore Item.css and Item.js since they are irrelevant to navbar and are not the working version (I only coded up the overall skeleton). Sorry for the confusion!